thrift-confignode/src/main/thrift/confignode.thrift

```diff
@@ -48,6 +58,11 @@ struct TDataNodeRegisterResp {
   4: optional TGlobalConfig globalConfig
 }
 
+struct TDataNodeRemoveResp {
```

Comment:
`TRemoveDataNodeResp`

Reply:
I keep it format with TDataNodeRegisterReq